idd/Energy+.idd.in

```diff
@@ -91976,8 +91976,128 @@ N6302,N6303,N6304,N6305,N6306,N6307,N6308,N6309,N6310,N6311,N6312,N6313,N6314,N6
 N6319,N6320,N6321,N6322,N6323,N6324,N6325,N6326,N6327,N6328,N6329,N6330,N6331,N6332,N6333,N6334,N6335,  \note fields as indicated
 N6336,N6337,N6338,N6339,N6340,N6341,N6342,N6343,N6344,N6345,N6346,N6347,N6348,N6349,N6350,N6351,N6352,  \note fields as indicated
 N6353,N6354,N6355,N6356,N6357,N6358,N6359,N6360,N6361,N6362,N6363,N6364,N6365,N6366;  \note fields as indicated
+
+Table:Lookup,
+       \memo Lookup tables are used in place of curves and can represent any number
+       \memo of independent variables (defined as Table:IndependentVariable objects
+       \memo in a Table:IndependentVariableList). Output values are interpolated
+       \memo within the bounds defined by each independent variable and extrapolated
+       \memo beyond the bounds according to the interpolation/extrapolation methods
+       \memo defined by each independent variable.
+       \extensible:1
+  A1,  \field Name
+       \required-field
+       \reference AllCurves
+       \reference UniVariateTables
+       \reference BiVariateTables
+       \reference MultiVariateTables
+       \type alpha
+  A2,  \field Independent Variable List Name
+       \required-field
+       \type object-list
+       \object-list IndependentVariableListName
+  A3,  \field Normalize Output
+       \type choice
+       \key Yes
+       \key No
+       \default No
+  N1,  \field Minimum Output
+       \type real
+       \unitsBasedOnField A4
+  N2,  \field Maximum Output
+       \type real
+       \unitsBasedOnField A4
+  A4,  \field Output Unit Type
+       \type choice
+       \key Dimensionless
+       \key Capacity
+       \key Power
+       \default Dimensionless
+  A5,  \field External File Name
+       \type alpha
+       \retaincase
+  N3,  \field External File Column Number
+       \type integer
+       \minimum 1
+  N4,  \field External File Row Number
```

Comment:
How about "External File *Starting* Row Number" or similar. 

Also, docs and IDD notes should make it clear that the data fields below are only used if there is no external file. (Elementary, I know, but still should be said.)

Reply:
Good catch.